packages/web/src/itinerary/api.ts

```diff
@@ -18,6 +18,8 @@ const API_ROUTES_ENDPOINT = API_BASE_URL + "/routes";
 export async function searchLocation(
     request: FindPlaceRequest
 ): Promise<FindPlaceFromTextResponse> {
+    request.locationbias = "circle:1610@37.3496, -121.939";
```

Comment:
I also tried point, and I didn't get different results but it was a small sample size and I don't know if it actually changes anything. I put in the circle just in case as that was mentioned in the API reference.